File: app/src/main/java/asvid/beercounter/CounterListListener.java
package asvid.beercounter;

import asvid.beercounter.data.CounterItem;

public interface CounterListListener {
    void onItemDelete(CounterItem item);

    void onItemClicked(CounterItem item);

    void onItemIncrement(CounterItem item);

    void onItemDecrement(CounterItem item);
}

File: app/src/main/java/asvid/beercounter/CounterListListener.kt
package asvid.beercounter

import asvid.beercounter.data.CounterItem

interface CounterListListener {
    fun onItemDelete(item: CounterItem, position: Int)

    fun onItemClicked(item: CounterItem, position: Int)

    fun onItemIncrement(item: CounterItem, position: Int)

    fun onItemDecrement(item: CounterItem, position: Int)
}
